matrix-api.c

```diff
@@ -98,18 +90,6 @@ void matrix_api_bad_response(MatrixConnectionData *ma, gpointer user_data,
 #define HEADER_PARSING_STATE_LAST_WAS_VALUE 0
 #define HEADER_PARSING_STATE_LAST_WAS_FIELD 1
 
-typedef struct {
```

Comment:
It is part of the api and it is needed so that the structs exist in other files that include the api header.